### PR TITLE
Vchuprakov

### PR DESCRIPTION
**Pull request by victorxup**
Task008 done

### CODE REVIEW

File: src/main/java/com/epam/university/java/core/task008/Task008Impl.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import java.util.Stack;
+
+/**
+ * Strings and braces.
+ */
+public class Task008Impl implements Task008 {
+    @Override
+    public boolean isValid(String sourceString) {
+        if (null == sourceString) return false;
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task008/Task008Impl.java]: New lines and braces! In any case!

Comment (victorxup) [reply src/main/java/com/epam/university/java/core/task008/Task008Impl.java]: i don't see the point,
what does it mean?..

Comment (victorxup) [reply src/main/java/com/epam/university/java/core/task008/Task008Impl.java]: do you mean, that verification for null is excessive?

Comment (aabarmin) [reply src/main/java/com/epam/university/java/core/task008/Task008Impl.java]: I said, that you should always add curly braces:

```
if (null == sourceString) {
    return false;
}
```

Comment (victorxup) [reply src/main/java/com/epam/university/java/core/task008/Task008Impl.java]: Got it

File: build.gradle [hunk 2]
```diff
@@ -2,13 +2,27 @@ group 'com.epam.university'
...
+    ignoreFailures = true
+    reports {
+        html.enabled = true
+        xml.enabled = false
+    }
+}
+
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    compileOnly 'org.projectlombok:lombok:1.16.18'
```
Comment (aabarmin) [inline build.gradle:27]: Do you really need lombok for our tasks?

**Comment by aabarmin** [comment]
There are a lot of style warnings in build log, could you fix it?

File: build.gradle (hunk 2 above)
Comment (victorxup) [reply build.gradle]: naah..
just addicted)

Comment (victorxup) [reply build.gradle:27]: naah..
just addicted)

**Comment by aabarmin** [comment]
There is a lot of style check errors. Please, fix them and I will merge your pull request.

**Pull request closed without merging**